README.md

```diff
@@ -78,7 +78,7 @@ Please make sure you’ve followed the guidelines in
     ```
 
   2. After you have cloned the EarlGrey repository, download all the dependencies using the
-  **setup-earlgrey.sh** (run the script from the cloned repo) script in the `Scripts/` directory.
+  [**setup-earlgrey.sh**](https://github.com/google/EarlGrey/blob/master/Scripts/setup-earlgrey.sh) (run the script from the cloned repo) script in the `Scripts/` directory.
```

Comment:
It's better to use a relative URL otherwise the link is invalid on forks & different branches.